phpcs.xml.dist

```diff
@@ -17,16 +17,29 @@
 		<!-- Allow short ternaries -->
 		<exclude name="WordPress.PHP.DisallowShortTernary.Found" />
 
+		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase" />
+		<exclude name="Universal.Files.SeparateFunctionsFromOO.Mixed" />
+		<exclude name="PEAR.NamingConventions.ValidClassName.StartWithCapital" />
+		<exclude name="PEAR.NamingConventions.ValidClassName.Invalid" />
+
+		<exclude name="WordPressVIPMinimum.Hooks.AlwaysReturnInFilter.MissingReturnStatement" />
+
 		<!-- Generates too many false positives -->
 		<exclude name="WordPress.WP.CapitalPDangit.Misspelled" />
 		<!-- We use trigger_error extensively -->
 		<exclude name="WordPress.PHP.DevelopmentFunctions.error_log_trigger_error" />
+
+		<exclude name="WordPress.WP.I18n.InterpolatedVariableSingular" />
+		<exclude name="WordPress.WP.I18n.InterpolatedVariablePlural" />
+		<exclude name="WordPress.WP.I18n.MissingSingularPlaceholder" />
+		<exclude name="WordPress.WP.I18n.NonSingularStringLiteralText" />
+
+		<exclude name="Squiz.PHP.DisallowMultipleAssignments.FoundInControlStructure" />
```

Comment:
What is the rationale behind all these additional exclusions? Some of them, like `PEAR.NamingConventions.ValidClassName.StartWithCapital`, are pretty standard practise IMO. 
Maybe some explanatory comments will help here too?

Reply:
Good point, I just pushed up some comments for that so it's easier to understand. 

The tl;dr is that its for legacy reasons as we can't alter the problematic areas without breaking something. I'm hoping overtime we can find alternatives so this gives us a middle ground.

Reply:
Good point, done in the latest commit